spec/support/request_helpers.rb

```diff
@@ -5,17 +5,18 @@ module FakeGitHub
 
 def github_request_headers
   {
-    'Accept'          => 'application/vnd.github.v3+json',
-    'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
-    'Content-Type'    => 'application/json',
-    'User-Agent'      => 'Octokit Ruby Gem 4.2.0'
+    'Accept'=>'application/vnd.github.v3+json',
+    'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+    'Authorization'=>'token validKeyAbcdfgh123',
```

Comment:
Ha, I was worried for a brief moment this was a real credential and then I drank some coffee